db/migrate/20180524190818_destroy_non_existing_git_hub_users.rb

```diff
@@ -0,0 +1,5 @@
+class DestroyNonExistingGitHubUsers < ActiveRecord::Migration[5.1]
+  def up
+    execute 'DELETE FROM users WHERE github_id IS NULL'
```

Comment:
Some considerations 🤔:

First, I think we may want to prevent this from running on development to avoid the issue you pointed out on the PR description. I worry this may cause unexpected behavior for existing developers who update their local copy of the app.

Wrapping the `execute` statement on a `unless Rails.env.development?` should do it, I think 👍

Second point: do these users have any relationships with other models in the database? If they do, maybe we should consider doing this using Ruby so the destroy callbacks run. `User.where(github_id: nil).destroy_all`. I don't have access to production data to see if this is the case or not.

Reply:
1) I like the ruby destroy better anyway. 
2) Are you sure about not running the migration in dev env? I'd think that we want the user objects in the local database not to be different from the test and production databases?

Reply:
Then maybe we could generate `github_id`s for users without one in development to make them be compatible without deleting them?

Reply:
I intended to add a 'how-to' to the readme or troubleshooting guide. Or should I add it to the migration? Add a part to add gh_ids if env == dev?

Sorry, I missed this comment yesterday.